src/helpers/moveFile.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs-extra');
+
+async function moveFile(oldPath, newPath) {
+  await fs.move(oldPath, newPath, function (err) {
```

Comment:
fs.move can't be in one time be as `promise` and have `callback in 3rd argument` 

you should use promise style or callback style, not both

Reply:
corrected